### PR TITLE
remove all checks.d optionals requirements

### PR DESCRIPTION
**Pull request by degemer**
Only keep the core agent optional requirements.

### CODE REVIEW

File: requirements-opt.txt [hunk 1]
```diff
@@ -14,42 +14,8 @@
...
-
-# checks.d/pgbouncer.py
-# Require libpq
-psycopg2==2.6.2
-
-# checks.d/win32_event_log.py
-# checks.d/wmi.py
-# It's a pure python module, it doesn't require anything to install,
-# but needs the pywin32 extension to work
-wmi==1.4.9
```
Comment (truthbk) [inline requirements-opt.txt]: Don't wee need this for the windows system checks? https://github.com/DataDog/dd-agent/blob/master/checks/system/win32.py#L16

Comment (gmmeyer) [inline requirements-opt.txt]: I believe this is right.

Comment (degemer) [inline requirements-opt.txt]: I'm not sure, I checked and it seems we always use our internal implementation. We don't have any 'import wmi' or 'from wmi' in our code. cc @yannmh

Comment (yannmh) [inline requirements-opt.txt]: 👍

Comment (truthbk) [inline requirements-opt.txt]: Yeah, just for posterity, the internal checks.libs.wmi.WMISampler relies on:
```
import pythoncom
from win32com.client import Dispatch
```

**Pull request merged**